torchvision/prototype/transforms/_misc.py

```diff
@@ -142,18 +142,19 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return F.gaussian_blur(inpt, self.kernel_size, **params)
 
 
+def _default_arg(value: Any) -> Any:
+    return value
```

Comment:
Since we now having this idiom multiple times, should we have this in `_utils`? Maybe even hide this ugly construct in a 

```py
def _get_defaultdict(default: T) -> Dict[Any, T]:
    return defaultdict(functools.partial(_default_arg, default))
```

Otherwise we need to add an explanation comment to every occurrence of this idiom since it is not obvious what is happening and the convolution is needed.